Buildscripts/Flashing/merge.ps1

```
@@ -0,0 +1,20 @@
+param(
+    $port
+)
+
+if ((Get-Command "esptool" -ErrorAction SilentlyContinue) -eq $null)
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)